### PR TITLE
Dev interlaced homepage

### PR DESCRIPTION
**Pull request by EarthenSky**
Make homepage png smaller (4MB now) & interlaced (fancy fast loading \:])

see https://nuwen.net/png.html

or, try loading [this image](https://raw.githubusercontent.com/CSSS/csss-site-frontend/91ccb251c029ccca547fddadc2c5740c7d868d3e/public/static/files/main/homepage-smol.png) (Ctrl + Shift + R)

### CODE REVIEW

File: src/main/pages/Home.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const Home = () => {
     <Page>
       <div className="relative w-full" style={{ height: 'calc(100vh - 32px)' }}>
         <animated.img
-          src="/static/files/main/homepage.jpg"
+          src="/static/files/main/homepage-smol.png"
```
Comment (micahdbak) [inline src/main/pages/Home.js:70]: Can you rename this file to "mural.png"?

If you do this I'll approve!!

**Comment by micahdbak** [comment]
@EarthenSky - poke poke

**Comment by EarthenSky** [comment]
No progress yet (been working on the fe for admin page :sob:)

**Comment by micahdbak** [comment]
Please do a `git mv` and push and ill approve the PR 🥺

File: src/main/pages/Home.js (hunk 1 above)
Comment (EarthenSky) [reply src/main/pages/Home.js:70]: https://github.com/CSSS/csss-site-frontend/pull/57/commits/be101ae17062018812b60dcff5bc3009ef3812b0

Comment (micahdbak) [reply src/main/pages/Home.js:70]: update dis line >:(

**Review by micahdbak — APPROVED** [review]
Update the src attribute in Home.js and then merge please!

**Pull request merged**